src/common/constants/app.constants.ts

```diff
@@ -1,5 +1,8 @@
-export const DATE_FORMAT = 'YYYY-MM-DD';
 export const ROLES = {
   ADMIN: 'admin',
   USER: 'user',
 };
+export const magicLinkExpirySeconds = 3600;
+export const jwtExpiresDefaultSeconds = 3600;
+export const emailFromEnvKey = 'EMAIL_FROM';
+export const backendUrlEnvKey = 'BACKEND_URL';
```

Comment:
hmmm I dont understand what is it and why it needed. if its env key, no need to have it in constant files